### ISSUE TITLE
BANGKIL

### ISSUE THREAD

**Issue by johnbent** [tekinged/missing#35]
Synonym for kalbasang. Confirm?

**Comment by s-o-u-a-n-g** [comment]
Yes. Just the same name for pumpkin/squash from different colonial languages (English & Spanish). 

souang
